lib/resty/etcd/v3.lua

```diff
@@ -1076,55 +1083,26 @@ function _M.timetolive(self, id, keys)
 end
 
 function _M.leases(self)
-    local endpoint, err = choose_endpoint(self)
-    if not endpoint then
-        return nil, err
-    end
-    return _request_uri(self, endpoint, "POST",
-                        endpoint.full_prefix .. "/lease/leases")
+    return _request_uri(self, "POST", "/lease/leases")
 end
 
 
 -- /version
 function _M.version(self)
-    local endpoint, err = choose_endpoint(self)
-    if not endpoint then
-        return nil, err
-    end
-    return _request_uri(self, endpoint, "GET",
-                        endpoint.http_host .. "/version",
-                        nil, self.timeout)
+    return _request_uri(self, "GET", "/version", nil, self.timeout)
 end
 
 -- /stats
 function _M.stats_leader(self)
-    local endpoint, err = choose_endpoint(self)
-    if not endpoint then
-        return nil, err
-    end
-    return _request_uri(self, endpoint, "GET",
-                        endpoint.http_host .. "/v2/stats/leader",
-                        nil, self.timeout)
+    return _request_uri(self, "GET", "/v2/stats/leader", nil, self.timeout)
 end
 
 function _M.stats_self(self)
-    local endpoint, err = choose_endpoint(self)
-    if not endpoint then
-        return nil, err
-    end
-    return _request_uri(self, endpoint, "GET",
-                        endpoint.http_host .. "/v2/stats/self",
-                        nil, self.timeout)
+    return _request_uri(self, "GET", "/v2/stats/self", nil, self.timeout)
```

Comment:
Not about this PR: is there `/v2` API in the `/v3` file?

Reply:
not for API exists in both version, like `/get` and `/put`